indico/htdocs/sass/partials/_core.scss

```diff
@@ -64,7 +64,7 @@
 }
 
 .weak-hidden {
-    display: none;
+    display: none !important;
```

Comment:
:bomb: This is going to break stuff! We have `.hidden` for this. weak-hidden is explicitly not important, hence "weak".